css/style.css

```diff
@@ -1,3 +1,943 @@
+  .product-mandel_b:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-mandel_bl {
+    position: absolute;
+    display: inline-block;
+    right: 42px;
+    bottom: 17px;
+    background: #41479b;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-mandel_bl:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-mandel_y {
+    position: absolute;
+    display: inline-block;
+    right: 31px;
+    bottom: 17px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-mandel_y:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  /* cloud */
+
+  .products__product-cloud {
+    display: block;
+    align-self: flex-start;
+    text-align: left;
+    position: relative;
+    width: 208px;
+    height: 261px;
+  }
+
+  .product-cloud_g {
+    position: absolute;
+    display: inline-block;
+    right: 65px;
+    bottom: 17px;
+    background: #c4c4c4;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_g:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_b {
+    position: absolute;
+    display: inline-block;
+    right: 53px;
+    bottom: 17px;
+    background: #257da9;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_b:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_r {
+    position: absolute;
+    display: inline-block;
+    right: 42px;
+    bottom: 17px;
+    background: #b62e3c;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_r:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_y {
+    position: absolute;
+    display: inline-block;
+    right: 31px;
+    bottom: 17px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_y:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .products__img {
+    display: block;
+    width: 169.89px;
+    height: 173.07px;
+    margin: 0 auto 23.67px auto;
+  }
+
+  .products__name {
+    font-size: 12px;
+  }
+
+  .products__cost {
+    font-size: 14px;
+  }
+}
+
+/* desktop 1170 */
+
+@media all and (min-width: 1170px) {
+  .page {
+    max-width: 1170px;
+  }
+
+  .page__header {
+   max-width: 1170px;
+   margin: 0 auto 20.17px auto;
+}
+
+ .header__logo {
+   width: 55px;
+   height: 55px;
+}
+
+ .menu__search {
+   width: 177.93px;
+}
+
+ .search__img {
+   width: 13.52px;
+}
+
+ .cart__count {
+   width: 14.92px;
+   height: 14.92px;
+   padding-top: 2px;
+   border: 2px solid #000;
+}
+
+ .menu__flag {
+   width: 20px;
+   height: 13.09px;
+}
+
+ .new-collection__goto {
+   width: 1030px;
+   height: 280px;
+   margin: 0 auto 53.78px auto;
+ }
+
+ .new-collection__goto .goto__img {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)